dask_jobqueue/core.py

```diff
@@ -83,6 +85,8 @@
     scheduler_cls : type
         Changes the class of the used Dask Scheduler. Defaults to  Dask's
         :class:`distributed.Scheduler`.
+    shared_directory : str
```

Comment:
I thought of no better way to specify where to dump the files

Reply:
I am indifferent here.  ```temp_security_directory``` also sounds OK to me or maybe ```shared_temp_directory```, so there is a chance for reuse?

Reply:
OK for `shared_temp_directory`, but make it clear in the comment that this is currently used for writing temporary security files.